crates/runtime/src/flight/middleware/request_context.rs

```diff
@@ -97,6 +97,9 @@ where
                 Arc::clone(&request_context) as Arc<dyn AuthRequestContext + Send + Sync>
             );
 
-        Box::pin(async move { request_context.scope(inner.call(req)).await })
+        Box::pin(async move {
+            request_context.run_extensions().await;
```

Comment:
Are these extensions or middleware?

Reply:
Renaming this to `run_middleware` would work.

Reply:
initially that was `load_extensions`